CONTRIBUTORS

```diff
@@ -528,7 +528,7 @@ Thank you!
     trapexit <trapexit@spawn.link>
     Trever Adams <trever@middleearth.sapphiresunday.org>
     Tsantilas Christos <chtsanti@users.sourceforge.net>
-    <ture@turepalsson.se>
+    Ture Pålsson <ture@turepalsson.se>
```

Comment:
> Despite CONTRIBUTORS file being UTF-8

How is CONTRIBUTORS file different in "being UTF-8" aspect from regular source files? In other words, what makes CONTRIBUTORS UTF-8 that does not apply to regular source files?

AFAICT, there are no non-ASCII symbols in CONTRIBUTORS right now. In the past, many names in this file were converted from their native alphabet to ASCII. Should we preserve that practice or start accepting non-ASCII entries? To preserve, we can use this variation:

```suggestion
    Ture Palsson <ture@turepalsson.se>
```

Reply:
FTR; the Squid sources are also UTF-8. However we code in C/C++ with American language documentation.

"being UTF-8" means manual ASCII translation such as required by reviewer are not needed.

Reply:
> "being UTF-8" means manual ASCII translation such as required by reviewer are not needed.

I did not require anything. I did note an inconsistency or a change in practice (and suggested a way to avoid it if we want to avoid it).

Reply:
Moreover, that practice was explicitly stated in commit 6e541c01: "Non-ASCII entries are still banned (for now) because properly supporting them requires a serious dedicated effort." I suspect we still do not properly support non-ASCII entries (i.e. that "serious dedicated effort" has not been made). Thus, using an ASCII variation is the best way forward here.

Reply:
Thanks for that reference. So ... "policy" definitions buried in long documentation texts, set late on a weekend, then merged without feedback or any kind from the rest of the team. Said actions by @rousskov, in a PR and commit attributed to @yadij. Interesting.